test/NVQPP/mapping_test-1.cpp

```diff
@@ -6,9 +6,11 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: nvq++ %cpp_std -v %s -o %t --target oqc --emulate && CUDAQ_DUMP_JIT_IR=1 %t |& FileCheck %s
+// REQUIRES: c++20
```

Comment:
There is clearly an issue with using C++17 here - I believe the error, if I remember correctly - was a failure to lower. 
There might even be the suggestion in the message to file a bug report. This needs to be fixed, not permanently disabled.

Reply:
Interesting enough, this test now actually passes and no longer fails with C++17 - one of the more recent PRs seems to have fixed it.

Reply:
https://github.com/NVIDIA/cuda-quantum/pull/1154/commits/62a59874eabbc8442a2cdc5c22a21d5096b0356b